packages/api-logs/main.css

```diff
@@ -104,6 +104,15 @@
 .hub-reference .logs .logs-list table th.time {
   width: 15%;
 }
+.hub-reference .logs .logs-list table td.useragent {
+  display: flex;
```

Comment:
In general, we probably shouldn't be overriding the built-in  display of natural HTML elements, like forcing this `<td>` to display as a `flex` box. (Though I understand the reasoning behind your choice, for sure!) Hardly a showstopping issue, but if I had my druthers, I'd probably to wrap this in an extra `<span>` that I could display as an `inline-flex` box.

Reply:
sure, works for me 👍